extensions/vscode/webviews/homeView/package.json

```diff
@@ -8,7 +8,7 @@
     "test": "TZ=UTC vitest"
   },
   "dependencies": {
-    "axios": "^1.7.4",
+    "axios": "1.7.4",
```

Comment:
Axios is misbehaving with types right now. We need to address it, but have kept it separate. I'm changing it to be an exact version here to avoid any minor version changes from hitting us now and in the future.